src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/wataskconfigurationapi/services/RoleAssignmentServiceApiTest.java

```diff
@@ -83,6 +84,51 @@ void queryRoleAssignmentTest() throws IOException {
         assertThat(roleAssignmentResource.getRoleAssignmentResponse().get(0)).isEqualTo(expectedRoleAssignment);
     }
 
+
+    @Test
+    void queryRoleAssignmentAndReceiveNewRolesTest() {
```

Comment:
Maybe we could have a test that receives all the role assignments in a list?  Don't think it would affect anything, just for coverage.

Reply:
That would avoid the need of having to handle the exception inside the loop thanks